wiki/People/The_Team/ru.md

```diff
@@ -1,56 +1,95 @@
 ---
-outdated: true
+tags:
+  - стафф
+  - команда
```

Comment:
```suggestion
tags:
  - osu! staff
  - osu!team
  - osu! team
  - staff
  - team osu!
  - стафф
  - команда
```

Reply:
do we really need these in translations? if yes, it's worth adding a sentence or two in ASC regarding that

Reply:
(also, if yes, feel free to commit the suggestion yourself)

Reply:
afaik they are needed, but even if not, theres no harm in adding